java/lang/os_command_injection/testdata/bad.java

```diff
@@ -0,0 +1,20 @@
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class Foo extends HttpServlet {
+    public void badExec(HttpServletRequest request, HttpServletResponse response) {
+        String name = request.getParameter("name");
+
+        Runtime.exec("sh -c echo " + name);
```

Comment:
I think this is how it works:

```suggestion
        Runtime.getRuntime().exec("sh -c echo " + name);
```